client/src/components/PastaShow.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+
+const PastaShow = (props) => {
+    const [pasta, setPasta] = useState({
```

Comment:
This file looks like it has a tab indentation size of 4, whereas other files have a tab size of 2. This should be consistent across a codebase, you can sync with your group to ensure your VS code settings are the same for tab size